components/CommonFooter.tsx

```diff
@@ -0,0 +1,196 @@
+      <a href="https://www.linkedin.com/company/open-seattle/">
+        <LinkedInIcon
+          sx={{ color: theme.palette.primary.contrastText }}
+          fontSize="large"
+        />
+      </a>
+    </GridItem>
+
+    <GridItem md={3}>
+      <SubText>Illustration by: Kylie Melendez</SubText>
+      <FooterItemTitle>@ Open Seattle 2023</FooterItemTitle>
+    </GridItem>
+
+    <GridItem md={4}>
+      <FooterItemTitle>Support us</FooterItemTitle>
+      <FooterSubItem>
+        <a href="mailto:name@email.com">How can you help</a>
+      </FooterSubItem>
+    </GridItem>
+
+    <GridItem md={4}>
+      <FooterItemTitle>About</FooterItemTitle>
+      <LinkSubItem url="/privacy-policy" name="Privacy Policy" />
+    </GridItem>
+
+    <GridItem md={1} display="flex">
+      <a href="https://github.com/openseattle">
+        <GitHubIcon
+          sx={{ color: theme.palette.primary.contrastText }}
+          fontSize="large"
+        />
+      </a>
+    </GridItem>
+  </Grid>
+)
+
+const CommonFooterSmallScreen = () => (
+  <Grid container spacing={5} direction="row">
+    {/* LOGO */}
+    <GridItem xs={5}>
+      <img
+        src={OSLogo.src}
+        style={{
+          height: '80%',
+          width: '120px',
+        }}
+      />
+    </GridItem>
+
+    <GridItem xs={5}>
+      <FooterItemTitle>Work with us</FooterItemTitle>
+      <LinkSubItem url="/volunteers_new" name="Volunteers" />
+      <LinkSubItem url="/partners_new" name="Partners" />
+    </GridItem>
+
+    <GridItem xs={2} display="flex">
+      <></>
+    </GridItem>
+
+    <GridItem xs={5}>
+      <FooterItemTitle>Contact us</FooterItemTitle>
+      <FooterSubItem>
+        <a href="mailto:name@email.com">info@openseattle.org</a>
+      </FooterSubItem>
+    </GridItem>
+
+    <GridItem xs={5}>
+      <FooterItemTitle>Support us</FooterItemTitle>
+      <FooterSubItem>
+        <a href="mailto:name@email.com">How can you help</a>
+      </FooterSubItem>
+    </GridItem>
+
+    <GridItem xs={2} display="flex">
+      <a href="https://www.linkedin.com/company/open-seattle/">
+        <LinkedInIcon
+          sx={{ color: theme.palette.primary.contrastText }}
+          fontSize="large"
+        />
+      </a>
+    </GridItem>
+
+    <GridItem xs={5}>
+      <SubText>Illustration by: Kylie Melendez</SubText>
+      <FooterItemTitle>@ Open Seattle 2023</FooterItemTitle>
+    </GridItem>
+
+    <GridItem xs={5}>
+      <FooterItemTitle>About</FooterItemTitle>
+      <LinkSubItem url="/privacy-policy" name="Privacy Policy" />
+    </GridItem>
+
+    <GridItem xs={2} display="flex">
+      <a href="https://github.com/openseattle">
+        <GitHubIcon
+          sx={{ color: theme.palette.primary.contrastText }}
+          fontSize="large"
+        />
+      </a>
+    </GridItem>
+  </Grid>
+)
+
+const CommonFooter = () => {
+  return (
+    <>
+      <Container
+        maxWidth="md"
+        sx={{ padding: '2rem 0rem', display: { md: 'flex', xs: 'none' } }}
```

Comment:
Should we use [MUI's media query](https://mui.com/material-ui/react-use-media-query/) here, so that react isn't creating all of the elements for the unused screen size in the virtual DOM?

Reply:
It's because it's also in a different order as well so the breakpoint wouldn't work... I thought it's easier to just recreate a different component.